src/ontology/uberon-edit.obo

```diff
@@ -214663,10 +214663,13 @@ property_value: provenance_notes "This class was sourced from an external ontolo
 
 [Term]
 id: UBERON:3011045
-name: gasserian ganglion
+name: obsolete gasserian ganglion
 synonym: "gasserian ganglia" RELATED [UBERON:3011045]
 is_a: UBERON:0000045 ! ganglion
 property_value: provenance_notes "This class was sourced from an external ontology (amphibian_anatomy). Its definitions, naming conventions and relationships may need to be checked for compatibility with uberon" xsd:string {source="http://purl.obolibrary.org/obo/aao.owl"}
```

Comment:
delete this `provenance_notes`

Reply:
I didn't add the provenance line. It has comment greases the way for the obsoletion and I kept it. I also thought I should go for the minimum changes. I will do it though no problem.